### DISCUSSION TITLE
Relative time not overwriting the time of the panel

### DISCUSSION THREAD

**Discussion by javiermanresa** [grafana/grafana#43136] [Dashboards, Panels & Visualizations]
Hi,

I'm getting what seems to be a bug when using the relative time (7 days) in a Stat visualization, since the calculations are made using the range time of the panel.

For this example, I'm using a 7 days relative time, but I'm getting the calculations for the last 24 hours. Even the graph seems to be created for the 7 days but only one portion (24 hours) is fully filled.

![image](https://user-images.githubusercontent.com/96175184/146399678-8e2f8e2d-47ac-4e46-9195-d10d2a119f22.png)

Thanks !

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
